canopy/design-system/src/styles/navigation.scss

```diff
@@ -44,10 +54,21 @@
     background: #eeeeee;
     width: 100%;
   }
+}
+
+@media only screen and (max-width: $mobile) {
+  .nav {
+    transition: width $transitionDuration-s linear, visibility 0s 0s;
+
+    .nav-logo {
+      width: 4rem;
+      height: 4rem;
+    }
 
-  &.open {
-    .nested {
-      height: fit-content;
+    &.hidden {
```

Comment:
are we using `hidden` for all elements we don't want to display? How does that work with `is-open`?

Reply:
updated